kstreams/test_utils/topics.py

```diff
@@ -115,9 +115,12 @@ def all_messages_consumed(cls) -> bool:
     @classmethod
     async def join(cls) -> None:
         """
-        Wait for all topic messages to be processed
+        Wait for all topic messages to be processed.
+        Only topics that have a consumer assigned should be awaited.
         """
-        await asyncio.gather(*[topic.join() for topic in cls.topics.values()])
+        await asyncio.gather(
+            *[topic.join() for topic in cls.topics.values() if not topic.consumed]
```

Comment:
shouldn't it be topic.consumer?

Reply:
It could be by there is a property checking that and whether the topic is empty or not https://github.com/kpn/kstreams/blob/master/kstreams/test_utils/topics.py#L48

Reply:
LGTM, I leave it to Reid